### PR TITLE
Inferno campaign - Devilish Plan

### PR DESCRIPTION
**Pull request by tomaas-zeman**
en-5
en-6
en-7

### CODE REVIEW

**Comment by qwrtln** [comment]
Would it be possible to use something like this as a starting tile?
inferno_1

**Comment by qwrtln** [comment]
Or this, with a little sharper edges.
inferno_sharp

**Comment by qwrtln** [comment]
Here's Rampart. If that works, we'll look for Dragon Utopia tile.
rampart_1

File: assets/images/inferno.png
Comment (qwrtln) [inline assets/images/inferno.png:1]: Isn't it a firewall? Inferno looks different.

Comment (tomaas-zeman) [reply assets/images/inferno.png:1]: inferno because inferno campaign :)

Comment (qwrtln) [reply assets/images/inferno.png:1]: I see. I think we should separate original graphics from the custom ones. The original ones should be named accordingly, not according to the scenario.

**Comment by qwrtln** [comment]
I just noticed that accidentally I cut out white color from tile numbers 😂

**Comment by qwrtln** [comment]
C1 tile is incorrect, as it looks like this: 
image

**Comment by qwrtln** [comment]
Ah, ignore me, it's C1, not #C1. Sadly, it's low-res:
c1

File: assets/images/inferno.png
Comment (tomaas-zeman) [reply assets/images/inferno.png:1]: OK

**Comment by tomaas-zeman** [comment]
Thanks for those HQ assets - I update the map

**Comment by qwrtln** [comment]
I'd say this poor last page needs some art.

**Comment by qwrtln** [comment]
Maybe the dogs? https://github.com/Heegu-sama/Homm3BG/blob/main/assets/art/cerberi.png

**Comment by tomaas-zeman** [comment]
> Maybe the dogs? https://github.com/Heegu-sama/Homm3BG/blob/main/assets/art/cerberi.png

Yes, dogs would look cool

**Review by qwrtln — APPROVED** [review]

**Comment by qwrtln** [comment]
po4a check

**Pull request merged**